src/include/verbose.h

```diff
@@ -134,6 +134,27 @@ struct __wt_verbose_multi_category {
 #define WT_VERBOSE_ISSET(session, category) \
     WT_VERBOSE_LEVEL_ISSET(session, category, WT_VERBOSE_LEVEL_DEFAULT)
 
+/* get the verbosity level for a given category. */
+#define WT_GET_VERBOSE_LEVEL(session, category, ret)                         \
+    do {                                                                     \
+        ret = S2C(session)->verbose[category];                               \
+    } while (0)
+
+/* Save the original level, and set new level  */
+#define WT_VERBOSE_SAVE(session, verbose_tmp, category, level)                \
+    do {                                                                      \
+        WT_GET_VERBOSE_LEVEL(session, category, ret);                         \
+        verbose_tmp[category] = ret;                                          \
+        WT_SET_VERBOSE_LEVEL(session, category, level);                       \
+    } while (0)
+
+/* Restore the original level  */
+#define WT_VERBOSE_RESTORE(session, verbose_tmp, category)                    \
+        do {                                                                  \
+            WT_GET_VERBOSE_LEVEL(session, category, ret);                     \
```

Comment:
`WT_GET_VERBOSE_LEVEL` doesn't seem to do anything here, please remove it.

This leaves us with a single use of `WT_GET_VERBOSE_LEVEL` in `WT_VERBOSE_SAVE`, so I'd recommend replacing it with `verbose_tmp[category] = S2C(session)->verbose[category];` and dropping the `WT_GET_VERBOSE_LEVEL` macro